ingenialink/ethercat/servo.py

```diff
@@ -169,7 +168,6 @@ def __init__(
         self.slave_id = slave_id
         self._connection_timeout = connection_timeout
         super(EthercatServo, self).__init__(slave_id, dictionary_path, servo_status_listener)
-        self._lock = threading.RLock()
```

Comment:
Undo that change too https://github.com/ingeniamc/ingenialink-python/commit/e9ea039ddc2000947cf9bbbd67d2f21903290d49#diff-353a4cd4aafad2effa546ece69ae67154a9a49a33881d267343cade73d8f05e3

Reply:
Done